tests/torch/test_nncf_network.py

```diff
@@ -493,22 +537,34 @@ def test_insertion_point_data_in_ip_nodes(self):
                 assert pre_hook_ip.target_node_name == nncf_node.node_name
 
     def test_operator_metatype_marking(self):
-        from nncf.torch.graph.operator_metatypes import PTConv2dMetatype, PTBatchNormMetatype, PTRELUMetatype, \
+        from nncf.torch.graph.operator_metatypes import PTBatchNormMetatype, PTRELUMetatype, \
             PTMaxPool2dMetatype, PTTransposeMetatype, \
             PTConvTranspose2dMetatype, PTDepthwiseConv2dSubtype, PTAddMetatype, PTAvgPool2dMetatype, PTLinearMetatype
         ref_scope_vs_metatype_dict = {
             "/" + MODEL_INPUT_OP_NAME + "_0": PTInputNoopMetatype,
-            "ModelForMetatypeTesting/NNCFConv2d[conv_regular]/conv2d_0": PTConv2dMetatype,
-            "ModelForMetatypeTesting/NNCFBatchNorm2d[bn]/batch_norm_0": PTBatchNormMetatype,
+            "ModelForMetatypeTesting/NNCFConv2d[conv_regular]/conv2d_0":
+                PTConv2dMetatype.module_metatype,
+            "ModelForMetatypeTesting/NNCFBatchNorm2d[bn]/batch_norm_0":
+                PTBatchNormMetatype.module_metatype,
```

Comment:
Shouldn't it be tested with registered user modules?
https://github.com/openvinotoolkit/nncf/pull/1430/files#r1048213174

Reply:
`test_operator_metatype_marking` is testing static NNCF modules metatypes (from `NNCF_MODULES_DICT`) and 
`test_custom_module_registering` is all about custom module, it checks that modules are being wrapped and all metatypes are correct